### ISSUE TITLE
Window frame too big on Wayland, even more after scaling

### ISSUE THREAD

**Issue by Abdillah** [electron/electron#22778]
This is actually application end-user bug, but it doesn't fit to report this on per-app basis because all app I have tried are affected.

**Description:** 
On HiDPI screen, Electron apps already scaled its window frame (window bar and close button) to match the display screen size (my assumption). When I scale my desktop using Wayland scale factor, for example to 1.5 or 150%, the window frame will get extremely bigger. I have checked this in Terminus and Atom and [hokein's sample apps](https://github.com/hokein/electron-sample-apps). This scaling doesn't change even when using option such as `atom --high-dpi-support=1 --force-device-scale-factor=1`. Notice on the screenshot that the size of window frame already different from native application.

FYI. The initial zoom of the app's content actually also already scaled, but I will focus on the window frame because it can't be modified with option mentioned.

### Preflight Checklist

* [x] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for an issue that matches the one I want to file, without success.

### Issue Details

* **Electron Version:** v2.0.18 (atom build), v4.2.8 (independent)
* **Operating System:** NixOS 19.09
* **Last Known Working Electron version:** -

### Expected Behavior
Have window frame scaled only after operating system setting.

### Actual Behavior
It is already scaled then scaled again after operating system setting.

### To Reproduce

1. Setup GNOME desktop (or any linux desktop) with Wayland mode.
2. Open electron apps. Look at the size of the frame. Close the apps.
3. Scale the Wayland system to 150%.
4. Open electron apps. Look at the size of the frame. Close the apps.
5. Open the apps using `... --high-dpi-support=1 --force-device-scale-factor=1`.
6. Open electron apps. Look at the size of the frame. Close the apps.

### Screenshots
![Before scale](https://user-images.githubusercontent.com/3030950/77123172-f5729f80-6a71-11ea-86d7-ad8fe9362050.png)
![After scale](https://user-images.githubusercontent.com/3030950/77123179-f73c6300-6a71-11ea-8d2c-029ccbd45105.png)

**Comment by Abdillah** [comment]
This is actually chrome bug, filed [the issue here](https://bugs.chromium.org/p/chromium/issues/detail?id=1064162).

**Comment by electron-triage** [comment]
The Electron version reported on this issue is no longer supported. See our [supported versions documentation](https://www.electronjs.org/docs/tutorial/support#supported-versions).

**If you're still experiencing this issue on a supported version, please open a new issue with an updated repro - a [Fiddle](https://www.electronjs.org/fiddle) is very appreciated.**

Electron has a large issues backlog. To help our team prioritize, we're closing older issues and asking for new issues with updated repro steps if it affects a supported version. This helps sort what issues are still relevant and helps us fix them more quickly.

Thanks for your patience and understanding!
